pygmt/helpers/decorators.py

```diff
@@ -25,6 +25,14 @@
             *Required if this is the first plot command*.
             *projcode*\[*projparams*/]\ *width*.
             Select map :doc:`projection </projections/index>`.""",
+    "A": r"""
+        area_thresh : int, float, or str
```

Comment:
Oh wait, just noticed this. Should use 'or' as a separator.
```suggestion
        area_thresh : int or float or str
```